packages/frontend/src/components/VisualizationConfigs/ChartConfigPanel/Legend/ReferenceLine.tsx

```diff
@@ -76,12 +76,17 @@ const ReferenceLineValue: FC<ReferenceLineValueProps> = ({
     if (isCustomDimension(field)) return <></>;
     if (isDateItem(field)) {
         if (isDimension(field) && field.timeInterval) {
+            // Uses the current date if the provided value is invalid
+            const parsedDate = dayjs(value).isValid()
+                ? dayjs(value).toDate()
+                : new Date();
```

Comment:
could we do 
```suggestion
                : dayjs().toDate()
```
here?

Reply:
Sure